tasks/sonar_runner.js

```diff
@@ -44,7 +44,10 @@ module.exports = function (grunt) {
         var dryRun = options.dryRun;
         var done = this.async();
 
-        options.sonar.language = options.sonar.language || 'js';
```

Comment:
hmm adding the language option "multi" would increase readability but in my opinion it would break the consistency with the regular behaviour of the sonar runner configuration.

I can add "multi" as an option if you like?